### PR TITLE
Update delfin support matrix

### PR DESCRIPTION
**Pull request by joseph-v**
**What type of PR is this?**

Update delfin support matrix

> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/support-matrix/delfin/_index.md [hunk 1]
```diff
@@ -17,10 +17,15 @@ Delfin has standardized common API services.
 | Dell EMC                                            | VNX (Block)               | 4.30 or later                      | CLI                     | Block   | √                        | √            | √   | √          | √    | √    | NA    | NA    | NA          | NA    | √                    | √         | ×           | ×               | ×                   | 1\. Before connecting a Dell EMC VNX Block storage device, install Navisphere Secure CLI on delfin installed node.                                                                                      |   |
```
Comment (qinwang-murphy) [inline content/support-matrix/delfin/_index.md:17]: According to the code implementation, I find that the statistics of the vmax device are different from those of the unity device. The red part indicates the result after the change. Please check it.
image

Comment (qinwang-murphy) [inline content/support-matrix/delfin/_index.md:17]: According to the code implementation, I find that the statistics of the vmax and unity devices are slightly different. The red part indicates the result after the change. Please check it.
image

Comment (joseph-v) [reply content/support-matrix/delfin/_index.md:17]: Done. Thank you

**Review by wisererik — APPROVED** [review]
lgtm

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**